Common/Ubuntu/librealsense/d4xx.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Realsense D4xx Service
+After==multi-user.target
+StartLimitIntervalSec=0
+Conflicts=
+
+[Service]
+User=apsync
+EnvironmentFile=
```

Comment:
I really don't know what I'm talking about but I guess these lines are important?  I.e. "EnvironmentFile=" with no path is required.

Reply:
For this I am using the same format as the .service file in the RPi2 folder. I don't think it is required when there is no path, but we can just leave it there for consistency and to reduce ambiguity (either default path or a defined path)?

Reply:
ok!